### PR TITLE
[AUTO-MERGE] Merge main into develop via merge-main-to-develop

### PR DESCRIPTION
**Pull request by idanov**
A new change in main cannot be merged into develop as part of the regular sync job, hence this PR. Please resolve the conflicts manually, and make sure to obtain 2 approvals once the builds pass.

### IMPORTANT NOTICE

Please let CircleCI merge this PR automatically, with merge commit enabled.

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1256"></a>

### CODE REVIEW

File: docs/source/tutorial/tutorial_template.md [hunk 1]
```diff
@@ -50,26 +42,21 @@ wheel>=0.35, <0.37 # The reference implementation of the Python wheel packaging
...
-
-Add the following requirements to your `src/requirements.in` lock file:
+Edit your `src/requirements.txt` file to include the following lines:
 
 ```text
 kedro[pandas.CSVDataSet, pandas.ExcelDataSet, pandas.ParquetDataSet]==0.17.6   # Specify optional Kedro dependencies
-kedro-viz==4.1.1                                                               # Visualise your pipelines
-openpyxl==3.0.9                                                                # Use modern Excel engine (will not be required in 0.18.0)
+kedro-viz~=4.0                                                                 # Visualise your pipelines
+openpyxl>=3.0.6, <4.0                                                          # Use modern Excel engine (will not be required in 0.18.0)
```
Comment (AntonyMilneQB) [inline docs/source/tutorial/tutorial_template.md:52]:
```suggestion
```

File: docs/source/tutorial/tutorial_template.md [hunk 2]
```diff
@@ -50,26 +42,21 @@ wheel>=0.35, <0.37 # The reference implementation of the Python wheel packaging
...
 
-Then run the following command to re-compile your updated dependencies and install them into your environment:
+To install all the project-specific dependencies, navigate to the root directory of the project and run:
 
 ```bash
-kedro install --build-reqs
+pip install -r src/requirements.txt
 ```
 
 You can find out more about [how to work with project dependencies](../kedro_project_setup/dependencies.md) in the Kedro project documentation. In a [later step of this tutorial](./create_pipelines.md#update-dependencies), we will modify project's dependencies to illustrate how, once you have installed project-specific dependencies, you can update them.
```
Comment (AntonyMilneQB) [inline docs/source/tutorial/tutorial_template.md:62]:
```suggestion
You can find out more about [how to work with project dependencies](../04_kedro_project_setup/01_dependencies.md) in the Kedro project documentation.
```

File: docs/source/tutorial/visualise_pipeline.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 ## Install Kedro-Viz
 
-You can install Kedro-Viz by running:
+If you did not already install Kedro-Viz when you [installed the tutorial project dependencies](02_tutorial_template.md#install-dependencies) then you can do so now by running:
```
Comment (AntonyMilneQB) [inline docs/source/tutorial/visualise_pipeline.md:7]:
```suggestion
If you did not already install Kedro-Viz when you [installed the tutorial project dependencies](tutorial_template.md#install-dependencies) then you can do so now by running:
```

**Review by AntonyMilneQB — APPROVED** [review]
I've just made some modifications to the docs that will hopefully fix the linkchecker.

**Review by MerelTheisenQB — APPROVED** [review]

**Pull request merged**